store/sqlstore/channel_member_history_store.go

```diff
@@ -46,15 +49,18 @@ func (s SqlChannelMemberHistoryStore) LogJoinEvent(userId string, channelId stri
 }
 
 func (s SqlChannelMemberHistoryStore) LogLeaveEvent(userId string, channelId string, leaveTime int64) error {
-	query := `
-		UPDATE ChannelMemberHistory
-		SET LeaveTime = :LeaveTime
-		WHERE UserId = :UserId
-		AND ChannelId = :ChannelId
-		AND LeaveTime IS NULL`
-
-	params := map[string]interface{}{"UserId": userId, "ChannelId": channelId, "LeaveTime": leaveTime}
-	sqlResult, err := s.GetMaster().Exec(query, params)
+	query, params, err := s.getQueryBuilder().
+		Update("ChannelMemberHistory").
+		Set("LeaveTime", leaveTime).
+		Where(sq.And{
+			sq.Eq{"UserId": userId},
+			sq.Eq{"ChannelId": channelId},
+			sq.Eq{"LeaveTime": nil},
+		}).ToSql()
+	if err != nil {
+		return errors.Wrapf(err, "channel_member_history_to_sql")
```

Comment:
My apologies, you don't actually need `Wrapf` if you are not using any formatting args. Just `errors.Wrap` is sufficient. Sorry for the double work.

Reply:
Perfect!